playroom/snippets.tsx

```diff
@@ -1727,6 +1727,46 @@ const logoSnippets = [
     },
 ];
 
+const stackingGroupSnippets = [
+    {
+        name: 'Stacking Group',
+        code: `
+        <StackingGroup
```

Comment:
I don't like this api, this component should be used like this:
```ts
<StackingGroup stacked={false} maxItems={5} moreItemsStyle={{type: 'circle', size: 64}}>
    <Avatar size={64} src="https://source.unsplash.com/600x600/?face" />
    <Avatar size={64} src="https://source.unsplash.com/600x600/?face" />
    <Avatar size={64} src="https://source.unsplash.com/600x600/?face" />
    <Avatar size={64} src="https://source.unsplash.com/600x600/?face" />
    <Avatar size={64} src="https://source.unsplash.com/600x600/?face" />
    <Avatar size={64} src="https://source.unsplash.com/600x600/?face" />
</StackingGroup>
```

```ts
type Props = {
    stacked?: boolean;
    maxItems?: number;
    moreItemsStyle: {
        type: 'circle' | 'square';
        size: 64 | 32 | 24;
    };
    children: React.ReactNode;
}
```

Reply:
The size of moreItemsStyle could be open because you can use any size in the children component (always following the multiplies of 8)